conductor/submitter.py

```diff
@@ -496,7 +491,7 @@ def populateInstanceTypeCmbx(self):
         '''
 
         self.ui_instance_type_cmbx.clear()
-        for instance_type in sorted(self._instance_types.values(), key=operator.itemgetter("cores", "memory"), reverse=True):
+        for instance_type in sorted(self._instance_types.values(), key=operator.itemgetter("cores", "memory")):
```

Comment:
I could probably go either way on this, but from a user experience, I'm not sure if we want the lowest instances listed first.  

If we do that, the user is always forced to scroll down to an instance type that is more appropriate for their workload.  I could certainly be wrong, but I would assume most users that want to use conductor are going to want to use instance types that have a minimum of 16 cores.
![image](https://user-images.githubusercontent.com/10409070/64652221-2776bb00-d3d8-11e9-88a9-9444de37ad61.png)

On the other hand, if we sort with the highest instance types at the top...
![image](https://user-images.githubusercontent.com/10409070/64652302-5ee56780-d3d8-11e9-94f9-3c6089becc21.png)


I think either approach has its pros/cons, but figured I would raise the issue since it looks like you made an intentional decision to change the presentation.